test/e2e/multicast_test.go

```diff
@@ -600,25 +600,57 @@ func runTestMulticastBetweenPods(t *testing.T, data *TestData, mc multicastTestc
 		data.RunCommandFromPod(data.testNamespace, senderName, mcjoinContainerName, sendMulticastCommand)
 	}()
 
+	runIpCmd := func(nodeName string, cmd []string, outputFilters ...string) ([]string, error) {
+		antreaPodName, err := data.getAntreaPodOnNode(nodeName)
+		if err != nil {
+			return nil, err
+		}
+		stdout, _, err := data.RunCommandFromPod(antreaNamespace, antreaPodName, agentContainerName, cmd)
+		if err != nil {
+			return nil, err
+		}
+		res := make([]string, 0)
```

Comment:
Not sure rewriting this way is better.
```    res := make([]string, 0)
    outer:
    for _, line := range strings.Split(stdout, "\n") {
        for _, f := range outputFilters {
            if !strings.Contains(line, f) {
                continue outer
            }
        }
        res = append(res, line)
    }
```

Reply:
interesting, I didn't know that
do we usually try to avoid Go labels in the Antrea codebase? @tnqn what do you think? I don't have a strong opinion here (well, I don't like goto statements...)

Reply:
Yes, I think `goto` is generally not recommended unless it's very necessary. But I do use label in loop with switch, otherwise it's hard to break the loop in the following case: https://github.com/antrea-io/antrea/blob/12afd3f7ed057ac8d96cc069eac30b996509e812/pkg/agent/controller/networkpolicy/networkpolicy_controller.go#L931-L947

Reply:
Ok, no string preference for me, so I added @ceclinux's proposed change